src/main/java/com/gorlah/kappabot/rest/CommandService.java

```diff
@@ -1,45 +1,24 @@
 package com.gorlah.kappabot.rest;
 
-import com.gorlah.kappabot.command.CommandProcessor;
 import com.gorlah.kappabot.function.FunctionProcessor;
 import com.gorlah.kappabot.rest.model.CommandRequest;
 import lombok.RequiredArgsConstructor;
-import org.apache.commons.text.StringSubstitutor;
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.util.HashMap;
-import java.util.Map;
+import java.util.Objects;
 
 @Service
 @RestController
 @RequiredArgsConstructor
 public class CommandService {
 
-    @Value("${discord.command.prefix}")
-    private String commandPrefix;
-
-    private final CommandProcessor commandProcessor;
     private final FunctionProcessor functionProcessor;
 
     @PostMapping("/command/run")
     public String runCommand(@RequestBody CommandRequest request) {
-        if (request == null) {
```

Comment:
Since you're replacing this check anyway, it would probably be a good time to make it throw a 400 instead of a 500, which would more accurately convey the nature of why the request failed.